### ISSUE TITLE
Can't "make" on Mac OS

### ISSUE THREAD

**Issue by YaKalmar0** [sergiomsilva/alpr-unconstrained#134]
I get message `clang: fatal error: unsupported option '-fopenmp` after typing command `make` in darknet directory. I have already updated clang but I don't understand how to run it properly via `make`.
Thanks in advance!

**Comment by tailtq** [comment]
I also encountered this yesterday. I thought this Darknet is out of date or something and replaced it entirely with the newest version which was pulled from [here](https://github.com/pjreddie/darknet). Now it works really well

**Comment by YaKalmar0** [comment]
> I also encountered this yesterday. I thought this Darknet is out of date or something so I replaced it entirely with the newest version which was pulled from [here](https://github.com/pjreddie/darknet). Now it works really well

True, thank you!

**Comment by YaKalmar0** [comment]
> I also encountered this yesterday. I thought this Darknet is out of date or something so I replaced it entirely with the newest version which was pulled from [here](https://github.com/pjreddie/darknet). Now it works really well

Did you have an issue with `import darknet.python.darknet as dn
ImportError: No module named darknet.python.darknet`? If yes, how did you solve it?

**Comment by tailtq** [comment]
> > I also encountered this yesterday. I thought this Darknet is out of date or something so I replaced it entirely with the newest version which was pulled from [here](https://github.com/pjreddie/darknet). Now it works really well
> 
> Did you have an issue with `import darknet.python.darknet as dn ImportError: No module named darknet.python.darknet`? If yes, how did you solve it?

No, I didn't see it. Can you double-check your files and directories?

**Comment by qurbat** [comment]
> > I also encountered this yesterday. I thought this Darknet is out of date or something so I replaced it entirely with the newest version which was pulled from [here](https://github.com/pjreddie/darknet). Now it works really well
> 
> True, thank you!

I think this can be closed with this solution. It worked for me as well.
